common/utils/utils.go

```diff
@@ -182,7 +182,7 @@ func PreloadGCSFiles(tables []ManifestTable) ([]ManifestTable, error) {
 	return tables, nil
 }
 
-// GetProject returns the cloud project we should use for accessing Spanner.
+// GetProject returns the cloud project we should use by default.
```

Comment:
Better to complete the statement by stating : GetProject returns the cloud project we should use by default to create resources

Reply:
Done.